XCTest.xcodeproj/project.pbxproj

```diff
@@ -255,6 +299,24 @@
 /* End PBXLegacyTarget section */
 
 /* Begin PBXNativeTarget section */
+		19D66F931DE6F57E00C112C0 /* SwiftXCTest-iOS */ = {
```

Comment:
Sorry, I missed this before. I think what we're going to need here is to add `-framework SwiftFoundation` into the `Other Linker Flags` build setting, but conditionalized such that it only applies to `Any macOS SDK`. Similar to what is described [here](http://stackoverflow.com/a/40004807/1217794).